ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/BlockProposalUtil.java

```diff
@@ -59,15 +60,38 @@ public BeaconBlock createNewBlock(
       final SSZList<Deposit> deposits)
       throws StateTransitionException {
     final UnsignedLong newEpoch = compute_epoch_at_slot(newSlot);
+    return createNewBlock(
+        signer,
+        newSlot,
+        previousState,
+        parentBlockSigningRoot,
+        StartupUtil.get_eth1_data_stub(previousState, newEpoch),
```

Comment:
Should we cut this altogether and require callers to supply eth1data?

Reply:
Fair point.

Reply:
done.